core/src/main/java/com/backblaze/b2/json/B2JsonHandlerMap.java

```diff
@@ -232,10 +232,18 @@ private B2JsonHandlerMap(Map<Class<?>, B2JsonTypeHandler<?>> initialMapOrNull) {
     /*package*/ synchronized <T> B2JsonTypeHandler<T> getUninitializedHandler(Type type) throws B2JsonException {
         // TODO validate the type - make sure it's resolved.
 
+        // class Item {
+        //     private int itemNumber;
+        //     private String itemName;
+        // }
         if (type instanceof Class) {
             final Class<T> clazz = (Class<T>) type;
             return getUninitializedHandlerForClass(clazz);
         }
+
+        // class Dataset {
+        //     private List<Double> measurements;
+        // }
         if (type instanceof ParameterizedType) {
```

Comment:
The type here is from the field, right? It's the `List<Double>`?

Having the `class Dataset` in the comment above is a little confusing.